### ISSUE TITLE
Bad enum naming cause photon generation error in index.d.ts

### ISSUE THREAD

**Issue by typedef42** [prisma/prisma2#1185]
When using type-graphql, bad enum naming causes generation problems.
For example, for a model (@ObjectType()) named `User`, an enum named `enum UserArgs` will cause this kind of errors at run time :

```node_modules/@prisma/photon/index.d.ts:95:22 - error TS2451: Cannot redeclare block-scoped variable 'UserArgs'.```

I guess naming enum types like that is a poor naming choice, but it might be worth to put some naming restriction/warning in the docs?

reproduction :  https://github.com/typedef42/prisma2-test-enum

**Comment by pantharshit00** [comment]
Yes, I can confirm this issue. We do indeed name filter arguments as `<model_name>Args` and they are clashing with the name that your model has. 

I would actually confirm this as a bug rather than a documentation issue because it is very common to name enums like that.
